src/imagery/i.landsat/i.landsat.download/i.landsat.download.py

```diff
@@ -280,34 +270,21 @@ def main():
                     "option is provided, files will be downloaded in /tmp"
                 )
             )
-
         else:
-
-            ee = EarthExplorer(user, password)
-
-            for scene in sorted_scenes:
-
-                gs.message(_("Downloading scene <{}> ...").format(scene["entity_id"]))
-
-                ee.download(
-                    identifier=scene["entity_id"],
-                    output_dir=outdir,
-                    timeout=int(options["timeout"]),
-                )
-
-            ee.logout()
+            geojson_temp_dir = gs.tempdir()
+            geojson_temp_file = os.path.join(geojson_temp_dir, "search_result.geojson")
+            print(geojson_temp_file)
+            with open(geojson_temp_file, "w") as file:
+                file.write(json.dumps(scenes))
+            gs.run_command(
+                "i.eodag",
+                file=geojson_temp_file,
+                provider="planetary_computer",
+                output=outdir,
+            )
 
 
 if __name__ == "__main__":
     options, flags = gs.parser()
 
-    # lazy import
```

Comment:
Shouldn't we do lazy import of eodag here and tell users that both eodag and i.eodag need to be installed for i.landsat.download to work?